news_items/026_direct_link/index.html

```diff
@@ -0,0 +1,37 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ pre {
+     width: 90%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+</style>
+
+<p>
+  <a href="https://github.com/jonatanschroeder">Jonatan Schroeder</a> [INSERT NOTE ABOUT AUTHOR]. It is now easier to link to individual course instances and assessments on PrairieLearn. Instructors can now open the "Settings" menu for a course instance or assessment and find a URL that can be shared with students:
+</p>
+
+<p>
+  <img src="assessmentSettings.png">
```

Comment:
This image shows the Student Link to an `assessment_instance`, but I think the actual link should be to an `assessment`?

Reply:
Fixed